contracts/BondingETH.sol

```diff
@@ -38,14 +38,15 @@ contract BondingETH is ABonding {
             _gton,
             _sgton,
             _bondType
-        ){}
+        )
+        {}
 
      /* ========== RESTRICTED ========== */
 
     /**
      * Function issues bond to user by minting the NFT token for them.
      */
-    function mint(uint amount) public payable mintEnabled returns(uint id) {
+    function mint(uint amount) public payable mintEnabled isWhitelisted(amount) returns(uint id) {
```

Comment:
I think we should leave mintEnabled check since we might need to urgently stop the contract altogether and not rely on necessity to update all individual allocations in whitelist account